packages/instantsearch.js/README.md

```diff
@@ -118,16 +121,16 @@ yarn add algoliasearch@4 @algolia/client-search
 yarn add @types/algoliasearch@3
 ```
 
->v3.x is deprecated and will soon no longer be supported.
+> v3.x is deprecated and will soon no longer be supported.
 
 ## Documentation
 
 The documentation is available on the [Algolia website](https://www.algolia.com/doc/guides/building-search-ui/what-is-instantsearch/js/).
 
 ## Demos
 
-| E-commerce                                                                                                                                                                                                                   | Media                                                                                                                                                                                                         | Travel                                                                                                                                                                                                              |
```

Comment:
other changes are through running prettier. Can revert if needed but I think it's fine

Reply:
Let's keep them if it makes Prettier happy.